doc/api/crypto.markdown

```diff
@@ -768,6 +768,25 @@ default, set the `crypto.DEFAULT_ENCODING` field to 'binary'.  Note
 that new programs will probably expect buffers, so only use this as a
 temporary measure.
 
+## Caveats
+
+The crypto module still supports some algorithms which are already
+compromised. And the API also allows to use ciphers and hashes with a
```

Comment:
"And the API also allows the use of ciphers and hashes with a" (replace `to` with `the`)